schemas/bursar-export/filter/bursarExportFilterAge.json

```diff
@@ -0,0 +1,29 @@
+{
+  "description": "Filter by fees within certain days range",
+  "type": "object",
+  "properties": {
+    "type": {
+      "description": "Type of filter",
+      "type": "string",
+      "default": "Age"
+    },
+    "numDays": {
+      "description": "Number of days",
+      "type": "integer",
+      "minimum": 1
+    },
+    "condition": {
+      "description": "Condition to get days range",
+      "type": "string",
+      "enum": [
+        "LESS_THAN_EQUAL",
```

Comment:
just a question, does it make sense to create common enum for conditions? I see 3 schemas use the same set